packages/ui-components/package.json

```diff
@@ -0,0 +1,73 @@
+{
+    "name": "@sap-ux/ui-components",
+    "version": "0.0.1",
+    "license": "Apache-2.0",
+    "description": "SAP UI Components Library",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/SAP/open-ux-tools.git",
+        "directory": "packages/ui-components"
+    },
+    "bugs": {
+        "url": "https://github.com/SAP/open-ux-tools/issues?q=is%3Aopen+is%3Aissue+label%3Abug+label%3Aui-components"
+    },
+    "private": false,
+    "main": "dist/index.js",
+    "scripts": {
+        "clean:dist": "rimraf ./dist ./generators *.tsbuildinfo",
+        "clean": "rimraf ./storybook",
+        "build": "npm-run-all -l -s clean -p build:extension build:copyfiles build:storybook",
+        "build:extension": "tsc  --build ./tsconfig.build.json --pretty",
+        "build:copyfiles": "copyfiles --up 1 \"./src/**/*.scss\" ./dist/",
+        "lint": "eslint . --ext .ts,.tsx",
+        "lint:fix": "eslint --fix",
+        "lint:fix:all": "eslint . --ext .ts,.tsx --fix",
+        "format:fix": "prettier --write --ignore-path ../../.prettierignore",
+        "format:fix:all": "prettier --write '**/*.{css,scss,html,js,json,ts,tsx,yaml,yml}' '!**/{out,dist,typings,node_modules}/**' '!**/*.{svg,png,xml}' --ignore-path ../../.prettierignore",
+        "storybook": "start-storybook -s ./.storybook/static",
+        "build:storybook": "npm-run-all -l -p build:storybook:bundle build:storybook:compile",
+        "build:storybook:bundle": "yarn build-storybook -o ./storybook",
+        "build:storybook:compile": "tsc  --build ./tsconfig-test.json --pretty --dry",
+        "test": "jest --ci --forceExit --colors --silent",
+        "test-u": "jest --ci --forceExit --colors  --silent -u"
+    },
+    "devDependencies": {
+        "storybook-addon-turbo-build":"1.1.0",
+        "@babel/core": "7.14.8",
+        "@babel/helper-define-map": "7.14.5",
+        "@fluentui/react": "8.66.2",
+        "@storybook/builder-webpack5": "6.5.10",
+        "@storybook/manager-webpack5": "6.5.10",
+        "@storybook/react": "6.5.10",
+        "@types/enzyme": "3.10.5",
+        "@types/enzyme-adapter-react-16": "1.0.6",
+        "@types/react": "16.14.0",
+        "@types/react-dom": "16.8.5",
+        "@types/react-virtualized": "9.21.11",
+        "@types/sanitize-html": "2.3.2",
+        "@types/uuid": "3.4.7",
+        "babel-jest": "^24.8.0",
+        "babel-loader": "8.1.0",
+        "copyfiles": "2.1.1",
+        "css-loader": "^5.2.6",
+        "enzyme": "3.11.0",
+        "enzyme-adapter-react-16": "1.15.2",
+        "eslint": "8.21.0",
```

Comment:
`eslint`, `typescript` and friends are already listed as devDependencies in the root, please remove them

Reply:
when eslint and typscript are removed we get following error in `pnpm install`
<img width="781" alt="image" src="https://user-images.githubusercontent.com/103586139/188861796-91f031c9-e8d8-4b3a-851c-866ba2046cce.png">

Reply:
when eslint and typscript are removed we get following error in `pnpm install`
<img width="781" alt="image" src="https://user-images.githubusercontent.com/103586139/188861796-91f031c9-e8d8-4b3a-851c-866ba2046cce.png">

Reply:
ugh. Ok, then keep them but, please, make sure that the versions are the same as in the root `package.json`